internal/pkg/describe/service.go

```diff
@@ -280,6 +286,22 @@ func (d *appRunnerServiceDescriber) ServiceARN() (string, error) {
 	return "", fmt.Errorf("no App Runner Service in service stack")
 }
 
+func (d *appRunnerServiceDescriber) vpcIngressConnectionARN() (*string, error) {
+	serviceStackResources, err := d.ServiceStackResources()
```

Comment:
Ahh perhaps we should cache some information to reduce the amount of API calls. I see `IsPrivate` getting called a lot

Reply:
`ServiceStackResources` caches the list of resources :)